src/test/java/org/medibloc/panacea/GrpcAolTest.java

```diff
@@ -133,12 +135,33 @@ private void testAddRecord(MsgAddRecord msg) throws PanaceaApiException, IOExcep
                 msg,
                 memo,
                 fee,
-                BroadcastMode.BROADCAST_MODE_BLOCK);
+                BroadcastMode.BROADCAST_MODE_SYNC);  // Used the sync mode as an example
 
         TxResponse response = client.broadcast(request);
         System.out.println(response.toString());
         Assert.assertNotNull(response.getTxhash());
         Assert.assertEquals(0, response.getCode());
+
+        String txHash = response.getTxhash();
+
+        while (true) {
```

Comment:
Come to think of it, it's a test code, but it looks like a code that can fall into an infinite loop.
I think it would be good to add logic that fails after repeating 10 times.

Reply:
good point. improved it: 32e4352